auth/userModel.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+const userSchema = new mongoose.Schema({
+  username: {
+    type: String,
+    unique: true,
+    required: true,
+    lowercase: true
+  },
+  password: {
+    type: String,
+    required: true,
+    minlength: 4
```

Comment:
In the real world we're definitely going to want a longer password, minlength should _at least_ be 10.